### PR TITLE
[FIX] svg render on firefox

### LINKED ISSUE #7980: Sometimes Icons don't render on firefox on ubuntu
screenshot from 2017-08-30 10-52-20

### LINKED ISSUE #9179: Right toolbar not displayed in firefox ESR
Hi,

  I have upgraded from 0.58.4 to 0.59.4. Since then, the toolbar at right of the screen is not displayed in firefox ESR. I have tested in other versions of firefox, chrome and the standalone client and it is displayed, but not in firefox ESR. The space of the toolbar is displayed and buttons are there (and "usable") but the icons are not displayed.

screenshot

### PR DESCRIPTION
**Pull request by karlprieb**
@RocketChat/core 

Closes #7980 
Closes #9179

image

### CODE REVIEW

File: packages/rocketchat-ui/client/components/icon.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+/* globals isFirefox */
+
```
Comment (ggazzo) [inline packages/rocketchat-ui/client/components/icon.js]:
```javascript
const baseUrl = isFirefox ? () => window.location.href.replace(window.location.hash, '') : () => ""

Template.icon.helpers({ baseUrl })
```

Comment (rodrigok) [reply packages/rocketchat-ui/client/components/icon.js]: @ggazzo Why??

Comment (karlprieb) [reply packages/rocketchat-ui/client/components/icon.js]: So the isFirefox check run only one time.

Comment (ggazzo) [reply packages/rocketchat-ui/client/components/icon.js]: thanks @karlprieb

Comment (rodrigok) [reply packages/rocketchat-ui/client/components/icon.js]: The `if` will run only one time, but replace (much heavy) will continue running every time.

I don't see much performance improvement for such non readable code.

Comment (karlprieb) [reply packages/rocketchat-ui/client/components/icon.js]: That function will run over 50 times on every route change, thats why we shouldn't check if is firefox every time.

**Pull request merged**